app/windows/Storage/fileIntegration.js

```diff
@@ -1,10 +1,8 @@
-import { remote } from 'electron'
+import { dialog, app, shell } from 'electron'
 import Settings from 'electron-settings'
 import { addFilesFromFSPath, unpinObject } from '../../api'
 import DetailsWindow from '../Details/window'
 
-const { app, dialog, shell } = remote
```

Comment:
`remote` is `undefined` when is called from the main process (which is happening now since I used this in the menu), but importing `{ app, dialog, shell }` directly seems to work.